source/process/sg_general-guidelines.rst

```diff
@@ -0,0 +1,92 @@
+
+Use headings, lists, and emphasis
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Format text to give a visual hierarchy that allows readers to see the overall content of the page by scanning it. Use headings, lists, and emphasis to signal importance. Headings should summarize the topic of the underlying information so that when readers scan a page, they can get an accurate picture of the contents.
+
+Be clear
+  Write short, active sentences using everyday vocabulary. Maintain a visual separation between page elements.
+Be concise
+  Minimize content so it can be found and remembered. Keep pages short, modular, and focused on a single topic.
+Be consistent
+  Refer to one thing or idea with the same word throughout the page.
+
+Write for an International Audience
+-----------------------------------
+
+Mattermost documentation is translated into several languages for use in many countries and cultures. For this reason, you must consider cultural differences on a global scale. Names, places, events, and actions should be chosen as carefully as possible to avoid misunderstanding.
+
+Also remember that English is not the primary language of many readers of the documentation. Keep the following advice in mind when writing:
+
+- Choose words with one or very few meanings.
+- Use simple verb forms in writing. Most verbs in the simple form are likely to have an equivalent in another language.
+- Avoid jargon and slang words.
+- Limit difficult words to technical terms where their use is unavoidable.
+- Always make sure that your spelling is correct.
+
+Word Usage Guidelines
+---------------------
+
+To promote consistency and clarity, follow the word usage and spelling guidelines in the following list.
+
+login, log in
+  Use *login* as a noun or adjective, and *log in* as a verb. For example: *Use the login credentials that you received to log in to Mattermost.*
+
+setup, set up
+  Use *setup* as a noun or adjective, and *set up* as a verb. For example: *Set up your operating system as described in the Ubuntu documentation.*
+
+sign-in, sign in
+  Use *sign-in* as a noun or adjective, and *sign in* as a verb.
+
+single sign-on
+  Single sign-on is abbreviated as SSO. When using the long form in a heading with title case, it's *Single Sign-on*.
+
+can, might, may
+  The word *may* can have several meanings. To avoid ambiguity, use *can* or *might* instead of *may*. Use *can* to mean *capable of* and *might* to mean that something is possible. Use *may* only to give permission to do something.
+
+downtime
+  Use as one word *downtime*, not *down time*.
+
+Gender-neutral Text
+-------------------
+
+Avoid constructs where you are forced to write either *he or she* or *his or her*. You can use *they* or *their* as singular forms instead.
+
+Preferred
+  The community manager monitors the forum for well-written questions and answers, and posts them to the Contributors channel.
+Avoid
```

Comment:
I'm not sure I understand the difference between "avoid" and "do not use". If we want to avoid it, should it just be written in the preferred way